src/app/Modal/DeleteWarningModal.tsx

```diff
@@ -46,62 +46,58 @@ export interface DeleteWarningProps {
   visible: boolean;
   onAccept: () => void;
   onClose: () => void;
-  setShowDialog: () => void;
-  checkbox?: boolean;
-  setCheckbox?: React.Dispatch<React.SetStateAction<boolean>>;
+  disableDialog: () => void;
 }
 
-export const DeleteWarningModal = ({ warningType , items, visible, onAccept, onClose, setShowDialog, checkbox, setCheckbox}: DeleteWarningProps): JSX.Element => {
+export const DeleteWarningModal = ({ warningType , items, visible, onAccept, onClose, disableDialog}: DeleteWarningProps): JSX.Element => {
   const [doNotAsk, setDoNotAsk] = useState(false);
   const realWarningType = getFromWarningMap(warningType);
 
-  const description = `${realWarningType?.description}${(typeof items === 'undefined' || items.length <= 1) ?  "":"s"}: [${items?.join(", ")}]. ${realWarningType?.descriptionPartTwo}`
+  const description = 
+    <TextContent>
+      <Text>
+        {realWarningType?.description}{(typeof items === 'undefined' || items.length <= 1) ?  "":"s"}: [{items?.join(", ")}]
+        <br />       
+        {realWarningType?.descriptionPartTwo}
```

Comment:
Can we rename this property something other than `descriptionPartTwo`? Perhaps it can even just be part of the description, if it's possible to use things like `<br>` in the description text and have that render properly (not sure). If that doesn't work directly then rather than the `description` property being just a `string`, maybe it can be `JSX.Element` and you can pass in a `<Text>` element with the intended formatting.

Reply:
I don't mind how that last screenshot looks. The `items?.join` isn't *that* worth holding onto here, IMO. The whole `[thing1, thing2]` is kind of nice to have, but seems ultimately unnecessary given that this modal is only going to appear after the user has explicitly selected some item(s) and then clicked a Delete button. If they clicked a select-all checkbox on a very long table of items then this dialog's list becomes really long and unwieldy, too.

The last screenshot with the `: [sdfh, sdfhfds]` removed would still totally do the job, IMO.